pmd-core/src/main/java/net/sourceforge/pmd/benchmark/Benchmarker.java

```diff
@@ -37,9 +37,9 @@
 import net.sourceforge.pmd.util.datasource.DataSource;
 
 /**
- *
- *
+ * @deprecated use {@link TimeTracker} instead
  */
+@Deprecated
```

Comment:
This being a non-critical feature, I'd rather keep binary compatibility than force people to update (specially in a minor!). Actually, other than CLI, there is not a single integration of PMD that uses benchmarking, so the use case would still be minimal (non-existing even?), but just in case.